app/views/comment_mailer/send_notification_email.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to RubyGamedev, <%= @user.username %></h1>
```

Comment:
![email notification](https://cloud.githubusercontent.com/assets/11162514/9277772/f0daf0ee-4271-11e5-8517-3575c27b6bad.png)

We did already the design but it is not include in this PR